### PR TITLE
anyhow feature

### PR DESCRIPTION
**Pull request by i-user-link**

### CODE REVIEW

File: crates/napi/Cargo.toml [hunk 1]
```diff
@@ -53,6 +54,10 @@ once_cell = "1"
 thread_local = "1"
 bitflags = "1"
 
+[dependencies.anyhow]
+version = "1.0.61"
```
Comment (Brooooooklyn) [inline crates/napi/Cargo.toml:58]:
```suggestion
version = "1"
```

File: rustfmt.toml [hunk 2]
```diff
@@ -1,2 +1,10 @@
-tab_spaces = 2
 edition = "2021"
+tab_spaces = 2
```
Comment (Brooooooklyn) [inline rustfmt.toml:2]: I like this, but I want to improve the code styles after https://github.com/napi-rs/napi-rs/pull/1074

**Comment by i-user-link** [comment]
https://github.com/iuser-dev/i18n-db/blob/8c487c0fb0a61d35627c3a7867c68f35d0ccf057/src/i18n.rs#L25

**Comment by i-user-link** [comment]
配合这个宏用更简单
```rust
  macro_rules! ok {
    ($code:block) => {
      Ok((|| Ok::<_, anyhow::Error>($code?))()?)
    };
  }

  #[napi]
  impl Db {
    #[napi]
    pub fn trans(&self, src: String, to: String) -> Result<Option<Vec<u8>>> {
      ok!({
        let db = &self.db;
        let cf = &self.cf;
        let key = hashkey(to + &src);
        db.get_cf(&cf.trans, &key)
      })
    }
}
···

**Comment by Brooooooklyn** [comment]
Closed in https://github.com/napi-rs/napi-rs/pull/1275

**Pull request closed without merging**